erpnext/agriculture/doctype/harvest/harvest.json

```diff
@@ -80,12 +80,11 @@
    "reqd": 1
   },
   {
-   "allow_on_submit": 1,
    "default": "0",
-   "depends_on": "eval:doc.docstatus == 1",
+   "depends_on": "eval:doc.docstatus == 1\neval:doc.harvest_type !=\"Manicure\"",
    "fieldname": "is_finished",
    "fieldtype": "Check",
-   "label": "Is Finished"
+   "label": "Completely Harvested"
```

Comment:
this field can be removed since you have added this to Harvest Type

Reply:
field is removed